{{cookiecutter.github_repository}}/fabfile.py

```diff
@@ -46,11 +46,12 @@ def init(vagrant=False):
     local('npm install')
     local('npm run build')
     {%- endif %}
-    local('createdb %(project_name)s' % env)  # create postgres database
     {%- if cookiecutter.add_pre_commit.lower() == 'y' %}
     add_pre_commit()
     {%- endif %}
-    manage('migrate')
+    if not env.ci:
```

Comment:
I'm not sure how `env.ci` will populate, env is not system environment variable as far as I know.